scripts/setup.sh

```diff
@@ -13,6 +13,15 @@ else
   echo "Skipping language dependencies installation (asdf not found)"
 fi
 
+if command -v code >/dev/null; then
+  echo "Installing Artsy VSCode extensions..."
+  code --install-extension Artsy.artsy-studio-extension-pack
+  # TODO: Move this into artsy-studio-extension-pack
```

Comment:
I tried, but I can't figure out for the life of me how to authenticate properly, have tried and tried. I know @MounirDhahri was there recently, so looking for any help 🙏